src/backend/distributed/utils/shardinterval_utils.c

```diff
@@ -74,29 +74,31 @@ CompareShardIntervals(const void *leftElement, const void *rightElement,
 	/*
 	 * Left element should be treated as the greater element in case it doesn't
 	 * have min or max values.
-	 */
```

Comment:
I guess the function comment of `CompareShardIntervals ` should change slightly to reflect the changes. I'm OK to remove the second sentence and have something like `See the comments in the function for shard intervals with empty min/max values`

Reply:
Given that we explain the desired behavior in the function's top level comment, I went with removing the inline comment altogether

Reply:
Not that important in this case, just sharing my thoughts on this in general. Typically function comments become obsolete as we change the functions and we forget to update the function comments. So, I prefer to keep more detailed comments in the function instead of the top.